### ISSUE TITLE
Missing Language String on Error

### ISSUE THREAD

**Issue by kevincam3** [vdm-io/Joomla-Component-Builder#212]
I decided to test my Dynamic Get by unpublishing a user that I was querying for. It gave me the following error. See attached screenshot.
![image](https://user-images.githubusercontent.com/870934/34903903-2aa1788a-f822-11e7-8add-2ec3d3dbf383.png)

I searched for that language string and noticed that it was never created. Is that the correct behavior?

### System information (as much as possible)
- OS Name & Version: Windows 10
- MySql Version: 5.7.14
- Apache Version: 2.4
- PHP Version: 5.6.32
- Joomla Version: 3.8.3
- JCB Version: 2.6.9
- Browser: Chrome 63

**Comment by kevincam3** [comment]
So I did find the language file that has it, but it's in the admin language file and not the frontend language file.

![image](https://user-images.githubusercontent.com/870934/34903944-e8f6b0ca-f822-11e7-8f27-c93c66be7295.png)

**Comment by Llewellynvdm** [comment]
I will take a look at this, I thought that this was fixed. 

I picked this up many months ago.... my last conclusion was that this happens when your component default view redirects to a none component view. Then the component language file is not being loaded and therefore not available. There is not much I can do about that.

So are you 100% sure that the string is not in the front-end language file?
